packages/mdc-segmented-button/segment/_segment.scss

```diff
@@ -54,16 +75,44 @@
           segment-theme.$border-radius 0px;
       }
     }
+
+    .mdc-segmented-button__touch {
+      @include touch-target-mixins.touch-target($query: $query);
+    }
+
+    .mdc-segmented-button--touch {
+      @include touch-target-mixins.margin(
+        $component-height: 32px,
```

Comment:
Should the touch target height be the same or larger than the segment's height?

Reply:
By "touch target" would it actually affect the "touchable" area that the button has to register a click/touch? If that's the case, I would make the touch target be the same height as the segment so there are no "accidental" clicks/touches.

Reply:
I don't fully understand how the touch target is supposed to work, but I think it's meant to expand the touchable area for mobile or users who have trouble clicking accurately.

Reply:
Touch target height should be same as MDC button's touch target height (i.e., 48px)

Reply:
See https://material.io/develop/web/components/touchtarget